src/cryptonote_core/tx_pool.cpp

```diff
@@ -570,7 +505,6 @@ namespace cryptonote
       txpool_tx_meta_t meta;
       if (!m_blockchain.get_txpool_tx_meta(id, meta))
       {
-        MERROR("Failed to find tx_meta in txpool");
```

Comment:
Because it is now normal behavior to not actually find transaction meta info in the mempool when adding blocks to the main chain since we have the pool supplement. When you leave the line in, the console gets spammed with thousands of lines of "Failed to find tx_meta in txpool" despite it syncing correctly.

Reply:
I think it would actually be good to keep this `MERROR` log if you go with my other suggestion above to call `have_tx` before `take_tx`. This log was actually helpful in debugging another issue I found once